bodhi/server/util.py

```diff
@@ -713,8 +718,8 @@ def type2icon(context, kind):
     }.get(kind)
 
     return "<span class='label label-%s' data-toggle='tooltip'\
-            title='This is a %s update'><i class='fa fa-fw %s'></i></span> \
-            " % (cls, kind, fontawesome)
+            title='This is %s %s update'><i class='fa fa-fw %s'></i></span> \
+            " % (cls, kind_article, kind, fontawesome)
```

Comment:
Looks like we just need to add some test coverage to this to get CI to be happy. You can add the tests at ```bodhi.tests.server.test_util```. I think we could probably get by with two tests. Here's some untested handy-wavy pseudocode to get you started:

```
def test_consonant(self):
    """Test type2icon() with a kind that starts with a consonant."""
    self.assertEqual(util.type2icon('security'), 
                               ("<span class='label label-%s' data-toggle='tooltip'"
                                "title='This is a security update'><i class='fa fa-fw danger'></i></span> "))

def test_vowel(self):
    """Test type2icon() with a kind that starts with a vowel."""
    self.assertEqual(util.type2icon('enhancement'), 
                               ("<span class='label label-%s' data-toggle='tooltip'"
                                "title='This is an enhancement update'><i class='fa fa-fw success'></i></span> "))
```